index.Rmd

```diff
@@ -293,10 +293,22 @@ class(modelRF)
 str(modelRF)
 names(modelRF)
 
+# Inspect the prediction error
+modelRF$prediction.error
+
+# Calculate the overall accuracy
+1 - modelRF$prediction.error
+
 # Inspect the confusion matrix of the OOB error assessment
 modelRF$confusion.matrix
 ```
 
+The overall accuracy and the confusion matrix are often used to evaluate the results of a supervised classification. However, the confusion matrix can provide more detailed information by giving per-class accuracies.
+
+```{block, type="alert alert-info"}
+**Note**: If you wish to learn how to read a confusion matrix, check out this [tutorial](https://www.evidentlyai.com/classification-metrics/confusion-matrix).
+```
```

Comment:
I added a note to show students that knowing how to read a confusion matrix is optional. But I am not sure if that is right. Maybe Big Data expects confusion matrices to have been covered in Geoscripting. If that's the case, then we would need a paragraph or tutorial explaining how to interpret confusion matrices (which can pretty tricky to grasp). Let me know what you think.